docs/api/en/audio/AudioContext.html

```diff
@@ -29,7 +29,7 @@ <h3>[method:AudioContext getContext]()</h3>
 		otherwise set it to a new [link:https://developer.mozilla.org/en-US/docs/Web/API/AudioContext AudioContext].
 		</p>
 
-		<h3>[method:AudioContext setContext]( [param:AudioConetxt value] )</h3>
+		<h3>[method:AudioContext setContext]( [param:AudioContext value] )</h3>
```

Comment:
Can you please fix this issue for the Chinese version, too? 

https://github.com/mrdoob/three.js/blob/36524a4c6667e88e128f78d78b7a29acf80e46d8/docs/api/en/audio/AudioContext.html#L32

Reply:
Done.